InventoryManagement/src/main/java/com/ccs/inventorymanagement/route/UpdateClothingHandler.java

```diff
@@ -0,0 +1,98 @@
+package com.ccs.inventorymanagement.route;
+
+import com.ccs.inventorymanagement.config.RouteConfig;
+import com.ccs.inventorymanagement.domain.Clothing;
+import com.ccs.inventorymanagement.domain.Item;
+import com.ccs.inventorymanagement.service.ClothingService;
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.server.HandlerFunction;
+import org.springframework.web.reactive.function.server.ServerRequest;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+import java.util.function.Function;
+
+public class UpdateClothingHandler implements HandlerFunction<ServerResponse> {
+
+    private final ClothingService service;
+
+    public UpdateClothingHandler(ClothingService service) {
+        this.service = service;
+    }
+
+    @Override
+    public Mono<ServerResponse> handle(ServerRequest serverRequest) {
+        return serverRequest.bodyToMono(Request.class)
+                .flatMap(new Function<Request, Mono<? extends Clothing>>() {
```

Comment:
Make this a lambda and remove any casting

Reply:
Updated